pkg/arm/data/azuredeploy.json

```diff
@@ -185,7 +190,11 @@
                 "[resourceId('Microsoft.Network/loadBalancers', 'lb-apiserver')]"
 {{- end }}
 {{- if not IsUpgrade }},
-                "[resourceId('Microsoft.Network/networkSecurityGroups', 'nsg-{{ $app.Name | js }}')]"
+{{- if eq $app.Role "master" }}
+                "[resourceId('Microsoft.Network/networkSecurityGroups', 'nsg-master')]"
+{{- else }}
+                "[resourceId('Microsoft.Network/networkSecurityGroups', 'nsg-nonmaster')]"
```

Comment:
`s/nonmaster/worker/` ? this `nonmaster` sounds strange

Reply:
ok, will fix in follow up